File: SCMS/ManufacturerGUI/src/main/java/com/example/manufacturergui/model/Contract.java
package com.example.manufacturergui.model;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class Contract
{
    private Integer id;
    private String company;
    private String product;
    private Integer quantity;
    private String contractPath;

    public Contract(Integer id, String company, String product, Integer quantity, String contractPath)
    {
        this.id = id;
        this.company = company;
        this.product = product;
        this.quantity = quantity;
        this.contractPath = contractPath;
    }
}

File: SCMS/NodeClient/Data/Model/Contract.cs
namespace NodeClient.Data.Model;

public class Contract
{
    public int Id { get; init; }
    public int Product_id { get; init; }
    public string Customer_id { get; init; } = null!;
    public int Quantity { get; init; }
    public string Contract_path { get; init; } = null!;
}
